deploy/flux-installer/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v2
+name: installer
```

Comment:
should match the folder name, lets use `medulla-installer` instead as this chart installs all of medulla; not just flux.